src/main/steps/c100-rebuild/screening-questions/legal-representation/template.njk

```diff
@@ -0,0 +1,11 @@
+{% extends "common/template.njk" %}
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
+{% endblock %}
+{% block page_legalRepresentation %}
+<p class="govuk-body-m">
```

Comment:
These links alignment is more left intended than rest of the fields & title. 
Please check & fix .